release-notes-1.0.2.md

```diff
@@ -0,0 +1,7 @@
+# Overview to Ballerina 1.0.2
```

Comment:
Shouldn't this be 
```suggestion
# Overview of Ballerina 1.0.2
```

Noticed we have "to" in the previous release notes too, but shall we clarify?

Reply:
of sounds correct though. Let's go with that.